dotnetflix.Web/Pages/Tickets/TicketsPage.razor

```diff
@@ -21,8 +21,18 @@
                         var ticketType = TicketTypes.FirstOrDefault(tt => tt.Id == ticketOrder.TicketTypeId);
                         if (ticketType != null)
                         {
-                            var finalPrice = SelectedShow.BasePrice - ticketType.Discount;
+                            var basePrice = SelectedShow.BasePrice;
+                            
+                            // Set discount for sneak preview shows
+                            if (SelectedShow.SneakPreview)
```

Comment:
Deze hele page is dus herzien in de US19-SeatSelection branch, echter zie ik wat je hier doet en heb ik deze discount toegepast in de nieuwe logica. Deze is terug te vinden in de TicketsPage in de US19 branch line 125.

Reply:
De wijzigingen in deze page bij een merge conflict niet meenemen, of deze branch eerst mergen en dan US19 eroverheen.